### PR TITLE
[Tests-Only] Refactor `webUIInternalUsersToRoot` tests to not use skeleton files (part 2)

### PR DESCRIPTION
**Pull request by saw-jan**
## Description
Refactor  `webUIResharing` and `webUIResharingToRoot`  tests to not use skeleton files:
- webUIInternalUsersToRootPreviews
- webUIInternalUsersToRootShareWithPage
- webUIInternalUsersToRootSharingIndicator

## Related Issue
https://github.com/owncloud/QA/issues/659

## How Has This Been Tested?
- test environment:

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:
- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/webUISharingInternalUsersToRootSharingIndicator/shareWithUsers.feature [hunk 1]
```diff
@@ -85,28 +96,29 @@ Feature: Sharing files and folders with internal users
...
       | fileName   | expectedIndicators |
       | sub-folder | user-indirect      |
 
   @issue-2939
   Scenario: sharing indicator for user shares stays up to date
-    Given user "Carol" has been created with default attributes
-    And user "David" has been created with default attributes
+    Given user "Carol" has been created with default attributes and without skeleton files
+    And user "David" has been created with default attributes and without skeleton files
+    And user "Alice" has uploaded file "testavatar.png" to "simple-folder/testavatar.png"
```
Comment (kiranparajuli589) [inline tests/acceptance/features/webUISharingInternalUsersToRootSharingIndicator/shareWithUsers.feature:121]:
```suggestion
    And user "Alice" has uploaded file "testavatar.png" to "simple-folder/testimage.png"
```
will make a small diff

**Review by dpakach — COMMENTED** [review]
LGTM 👍🏼

File: .drone.star [hunk 2]
```diff
@@ -181,7 +181,7 @@ config = {
 					'webUISharingInternalGroupsToRootSharingIndicator',
 					'webUISharingInternalUsers',
 					'webUISharingInternalUsersCollaborator',
-					'webUISharingInternalUsersSharreWithPage',
+					'webUISharingInternalUsersShareWithPage',
```
Comment (individual-it) [inline .drone.star:184]: good catch, those tests never ran

**Review by individual-it — APPROVED** [review]

**Comment by individual-it** [comment]
@saw-jan please  solve the conflicts and merge

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14757/

<p>









</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14762/

<p>









</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIRename</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14762/

<p>







</p></details>

**Pull request merged**